Src/DependencyCollector/Shared/DependencyTrackingTelemetryModule.cs

```diff
@@ -31,12 +31,13 @@ public class DependencyTrackingTelemetryModule : ITelemetryModule, IDisposable
 
 #if NET45 || NETCORE
         private HttpCoreDiagnosticSourceListener httpCoreDiagnosticSourceListener;
+        private SqlClientDiagnosticSourceListener sqlClientDiagnosticSourceListener;
```

Comment:
If you mean, does full .net SQL Client publish DiagnosticSource events, then, no, it doesn't. This was enabled for both per feedback from @lmolkova